feconf.py

```diff
@@ -20,18 +20,24 @@
 import datetime
 import os
 
+
 # Whether to unconditionally log info messages.
 DEBUG = False
 
 # The platform for the storage backend. This is used in the model-switching
 # code in core/platform.
 PLATFORM = 'gae'
 
+# This should be string comparison, since all environment variables
+# are converted to string
+IS_MINIFIED = os.environ.get('MINIFICATION') == 'True'
+
 # Whether we should serve the development or production experience.
 if PLATFORM == 'gae':
     DEV_MODE = (
-        not os.environ.get('SERVER_SOFTWARE')
-        or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        (not os.environ.get('SERVER_SOFTWARE')
+         or os.environ['SERVER_SOFTWARE'].startswith('Development'))
+        and not IS_MINIFIED)
```

Comment:
It might prevent errors such as accidentally including numpy path in production, i.e, we check for only `DEV_MODE = true` to include it, right now we are also checking for `IS_MINIFIED`.